pkgs/servers/nosql/mongodb/mongodb.nix

```diff
@@ -48,15 +48,17 @@ let
 
   system-libraries = [
     "boost"
-    "pcre"
     "snappy"
     "yaml"
     "zlib"
     #"asio" -- XXX use package?
     #"stemmer"  -- not nice to package yet (no versioning, no makefile, no shared libs).
     #"valgrind" -- mongodb only requires valgrind.h, which is vendored in the source.
     #"wiredtiger"
-  ] ++ lib.optionals stdenv.isLinux [ "tcmalloc" ];
+  ] ++ lib.optionals stdenv.isLinux [ "tcmalloc" ]
+    ++ lib.optionals (lib.versionOlder version "7.0") [
+      "pcre"
+    ];
```

Comment:
Is our PCRE too old or too new or something?

Reply:
--use-system-pcre was removed, maybe pcre2 will work

Reply:
seems to build fine with pcre2